app/javascript/vue/forms/fields/repository_rows.vue

```diff
@@ -63,7 +63,7 @@ export default {
   },
   methods: {
     itemCardUrl(row) {
-      return repository_repository_row_path(row.repository_id, row.id);
+      return `${repository_repository_row_path(row.repository_id, row.id)}?form_repository_rows_field_value_id=${this.field.field_value.id}`;
```

Comment:
you can use here same syntax as in rails
`repository_repository_row_path({row.repository_id, row.id, {form_repository_rows_field_value_id: this.field.field_value.id})`